modules/gdscript/gdscript_parser.h

```diff
@@ -1300,6 +1300,7 @@ class GDScriptParser {
 		COMPLETION_PROPERTY_METHOD, // Property setter or getter (list available methods).
 		COMPLETION_RESOURCE_PATH, // For load/preload.
 		COMPLETION_SUBSCRIPT, // Inside id[|].
+		COMPLETION_SUPER, // super()
```

Comment:
```suggestion
		COMPLETION_SUPER, // super(), used for lookup.
```
It's more like an editing context but since everything still uses the term completion, I think we should make the purpose clear.